main.md

```diff
@@ -38,7 +38,7 @@
 
 ### Shallow cloning
 `$ git clone <link> --depth=1`
-- Git supports the notion of a “shallow clone”, which is a more succinctly meaningful way of describing a local repository with history truncated to a particular depth during the clone operation. By providing an argument of --depth 1 to the clone command, the process will copy only the latest revision of everything in the repository. This can be a lifesaver for Git servers that might otherwise be overwhelmed by CI/CD automation		
+- Git supports the notion of a “shallow clone”, which is a more succinctly meaningful way of describing a local repository with the history truncated to a particular depth during the clone operation. By providing an argument of --depth 1 to the clone command, the process will copy only the latest revision of everything in the repository. This can be a lifesaver for Git servers that might otherwise be overwhelmed by CI/CD automation		
```

Comment:
Can u make it short and concise in 1 line maybe..?

Reply:
Git supports the notion of a “shallow clone”, which is a better way of describing a local repository with the history truncated to a particular depth during the clone operation. By providing an argument of --depth 1 to the clone command, the process will copy only the latest revision of everything in the repository. This can be a lifesaver for Git servers that might otherwise be overwhelmed by CI/CD automation

Reply:
I have reduced two three words

Reply:
Is this ok

Reply:
> Can u make it short and concise in 1 line maybe..?

I think very few people know about shallow clone so this description is fine

Reply:
Something say 15-20 words max. Nobody is gonna read this long sentence

Reply:
@gkanishk replace this description and this PR is good to go